sections/06_discussion.md

```diff
@@ -137,3 +137,7 @@ interpretability problems, how can we best ensure reproducible models? What
 might a clinician, or policy maker, need to see in a deep model in order to
 influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation?*
+
+### Transfer learning/transferability of features
```

Comment:
As you point out, transfer learning and multi-task learning has come up as a theme in many papers and domains.  Do you suggest we defer most of that discussion in the domain-specific sections and coalesce it here?  Or introduce it in the specific sections and have a cross-domain recap here?